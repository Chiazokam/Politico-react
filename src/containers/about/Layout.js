import React from 'react';
import { Header, Footer, TextStrip } from '../../components/global';
import { Showcase } from '../../components/about';

const Layout = () => {
  const navItems = ['home', 'signup', 'login'];
  return (
    <React.Fragment>
      <Header navItems={navItems}/>
      <TextStrip>About Politico</TextStrip>
      <Showcase />
      <Footer />
    </React.Fragment>
    )
}

export default Layout;
