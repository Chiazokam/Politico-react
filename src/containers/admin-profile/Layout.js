import React, { Component } from 'react';
import { Header, Footer, PlainStrip } from '../../components/global';
import { Showcase } from '../../components/admin-profile';

class Layout extends Component {
  constructor(props) {
    super(props);
    this.state = {
      navItems: ['create a candidate', 'logout']
    };
  }

  render() {
  return (
    <React.Fragment>
      <Header navItems={this.state.navItems}/>
      <PlainStrip />
      <h2 className='user-profile-text'>Perform Admin Activities</h2>
      <Showcase />
      <Footer />
    </React.Fragment>
    )
  }
}

const mapStateToProps = state => ({
  message: state.user.message
});

export default Layout;
