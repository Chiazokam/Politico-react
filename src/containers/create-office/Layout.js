import React from 'react';
import { Header, Footer, PlainStrip } from '../../components/global';
import { CreateOffice } from '../../components/office';

const Layout = () => {
  const navItems = ['back', 'logout'];
  return (
    <React.Fragment>
      <Header navItems={navItems}/>
      <PlainStrip />
      <CreateOffice />
      <Footer />
    </React.Fragment>
    )
}

export default Layout;
