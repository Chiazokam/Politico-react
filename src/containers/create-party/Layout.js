import React from 'react';
import { Header, Footer, PlainStrip } from '../../components/global';
import { CreateParty } from '../../components/party';

const Layout = () => {
  const navItems = ['logout'];
  return (
    <React.Fragment>
      <Header navItems={navItems}/>
      <PlainStrip />
      <CreateParty />
      <Footer />
    </React.Fragment>
    )
}

export default Layout;
