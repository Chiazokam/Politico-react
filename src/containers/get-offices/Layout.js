import React from 'react';
import { Header, Footer, PlainStrip } from '../../components/global';
import { GetOffices } from '../../components/office';

const Layout = (props) => {
  const navItems = ['logout'];
  return (
    <React.Fragment>
      <Header navItems={navItems}/>
      <div id="vote-office">
        <PlainStrip />
        <GetOffices auth={props}/>
      </div>
      <Footer />
    </React.Fragment>
    )
}

export default Layout;
