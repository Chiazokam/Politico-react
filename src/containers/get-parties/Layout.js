import React from 'react';
import { Header, Footer, PlainStrip } from '../../components/global';
import { GetParties } from '../../components/party';

const Layout = () => {
  const navItems = ['logout'];
  return (
    <React.Fragment>
      <Header navItems={navItems}/>
      <div id="view-party">
        <PlainStrip />
        <GetParties />
      </div>
      <Footer />
    </React.Fragment>
    )
}

export default Layout;
