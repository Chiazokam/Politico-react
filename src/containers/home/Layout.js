import React from 'react';
import { Showcase } from '../../components/home';
import { Header, Footer } from '../../components/global';

const Layout = () => {
  const navItems = ['Home', 'About', 'Sign in'];
  return (
    <React.Fragment>
      <Header navItems={navItems}/>
      <Showcase />
      <Footer />
    </React.Fragment>
    )
}

export default Layout;
