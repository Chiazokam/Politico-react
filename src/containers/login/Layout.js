import React from 'react';
import { Header, Footer, PlainStrip } from '../../components/global';
import { Showcase } from '../../components/login';

const Layout = () => {
  const navItems = ['home', 'about', 'signup'];
  return (
    <React.Fragment>
      <Header navItems={navItems}/>
      <PlainStrip />
      <Showcase />
      <Footer />
    </React.Fragment>
    )
}

export default Layout;
