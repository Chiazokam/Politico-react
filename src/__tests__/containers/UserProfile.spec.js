import React from 'react';
import { shallow } from 'enzyme';
import { UserLayout } from '../../containers/user-profile/Layout';
import user from '../../constants/user.constants';

describe('<User Profile />', () => {
  global.localStorage.setItem('user', '{}')
  it('renders correctly', () => {
   
      const getCandidacy = jest.fn()
    
    const tree = shallow(<UserLayout getCandidacy={getCandidacy} />);
    expect(tree).toMatchSnapshot(); 
  });
});
