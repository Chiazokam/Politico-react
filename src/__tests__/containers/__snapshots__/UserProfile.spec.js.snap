// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<User Profile /> renders correctly 1`] = `ShallowWrapper {}`;
